adobe/spectrum-web-components
# docs: add webcomponents.dev demo link

## Description
Add link for webcomponents.dev demo to README.md:
https://studio.webcomponents.dev/edit/QO8mtFYZAFMQWqhavYob/src/index.ts?p=stories




## Related issue(s)



- https://github.com/adobe/spectrum-web-components/issues/4791

## Motivation and context



## How has this been tested?




    1. Go here
    2. Click on "Try it on WebComponents.dev" link
    3. See the demo



## Screenshots (if appropriate)

## Types of changes




## Checklist




## Best practices

This repository uses conventional commit syntax for each commit message; note that the GitHub UI does not use this by default so be cautious when accepting suggested changes. Avoid the "Update branch" button on the pull request and opt instead for rebasing your branch against `main`.
